translate/src/modules/editor/components/Editor.tsx

```diff
@@ -6,12 +6,17 @@ import './Editor.css';
 import { EditorMenu } from './EditorMenu';
 import { NewContributorTooltip } from './NewContributorTooltip';
 import { MachinerySourceIndicator } from './MachinerySourceIndicator';
+import { FailedChecks } from './FailedChecks';
+import { FailedChecksProvider } from '~/context/FailedChecksData';
 
 export const Editor = () => (
   <div className='editor'>
     <NewContributorTooltip />
     <TranslationForm />
     <MachinerySourceIndicator />
     <EditorMenu />
+    <FailedChecksProvider>
+      <FailedChecks />
+    </FailedChecksProvider>
```

Comment:
Have you verified that this fixes the problem? I'd be surprised if it does, because we already load the `FailedChecks` component in the `EditorMenu` element.